ongebo/fff-frontend
# #163673602 Enable users to register to Fast-Food-Fast

### What does this PR do?
Enables users to register to Fast-Food-Fast
#### Description of the task to be completed?
- Define action types for user registration
- Add Redux reducers for signup actions
- Update Redux root reducer
- Define signup action creators
- Add React components for user registration
- Install dependencies for testing Redux action creators
- Write tests
#### How should this be manually tested?
- Install all dependencies on this branch and run `yarn start`
- Navigate to localhost:3000/register in the browser
- Fill in the registration form to verify that it is working
#### Screenshot
![image](https://user-images.githubusercontent.com/34346070/52362741-454efc80-2a52-11e9-86bf-174941bf7978.png)

#### What are the relevant Pivotal Tracker stories?
[#163673602](https://www.pivotaltracker.com/story/show/163673602)
